src/formatters/source/pushResultFormatter.ts

```diff
@@ -46,6 +46,20 @@ export class PushResultFormatter extends ResultFormatter {
    * @returns a JSON formatted result matching the provided type.
    */
   public getJson(): PushResponse {
+    // throws a particular json structure.  commandName property will be appended by sfdxCommand when this throws
+    if (process.exitCode !== 0) {
```

Comment:
Is there a reason you are not using `!this.isSuccess()` here like you did above?

Reply:
isSuccess considers `68` (partial success) as a success but toolbelt didn't.  😢

Reply:
Ah, I see. Otherwise you couldn't perform a check below 👍